hotdoc/core/symbols.py

```diff
@@ -28,6 +28,8 @@ class Symbol(Base):
     display_name = Column(String)
     filename = Column(String)
     lineno = Column(Integer)
+    extent_start = Column(Integer)
```

Comment:
Hm indeed, there should be some renaming though, lineno should be explicitly decl_lineno or something :)